js_modules/dagster-ui/packages/ui-core/src/assets/AutoMaterializePolicyPage/PolicyEvaluationTable.tsx

```diff
@@ -155,6 +171,56 @@ const NewPolicyEvaluationTable = ({
             startTimestamp = evaluation.startTimestamp;
           }
 
+          const assetDisplayName = assetKey ? displayNameForAssetKey(assetKey) : '';
+          const lastEvaluationForAssetKey =
+            assetKey &&
+            lastEvaluationsByAssetKey &&
+            tokenForAssetKey(assetKey) in lastEvaluationsByAssetKey
+              ? lastEvaluationsByAssetKey[tokenForAssetKey(assetKey)]
+              : null;
+          const evaluationNodeRef =
+            assetKey &&
+            assetKeyPath &&
+            pushHistory &&
+            lastEvaluationForAssetKey &&
+            tokenForAssetKey(assetKey) !== tokenForAssetKey({path: assetKeyPath}) ? (
+              <>
+                [
+                <Tooltip
+                  content={
+                    lastEvaluationForAssetKey.evaluationId === evaluationId
+                      ? `Navigate to evaluation details for \`${assetDisplayName}\` for this tick`
+                      : `Navigate to evaluation details for \`${assetDisplayName}\` for a previous tick`
+                  }
+                >
+                  <a
```

Comment:
Anchors should be used when there is an href. You can use a `ButtonLink` or just a `Button` here instead.